subsys/net/lib/aws_jobs/src/aws_jobs.c

```diff
@@ -673,3 +675,31 @@ int aws_jobs_update_job_execution(struct mqtt_client *const client,
 
 	return mqtt_publish(client, &param);
 }
+
+#define JOB_ID_GET_PAYLOAD "{\"clientToken\": \"\"}"
+int aws_jobs_get_job_execution(struct mqtt_client *const client,
+				const char *job_id)
+{
+	char job_id_get_topic_buf[JOB_ID_GET_TOPIC_MAX_LEN + 1];
+	struct mqtt_topic job_id_get_topic;
+
```

Comment:
Should we check that `client` is not NULL?

Reply:
Probably :100: